<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autonomous</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>791</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Repo Path</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>590</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Build Options</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Executable</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Shared library</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Static library</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>690</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Build</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Toolchain Options</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Windows-x64-msvc</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>590</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Destination</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>C:\Repo1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabFiles">
     <attribute name="title">
      <string>Sources</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>761</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Files</string>
      </property>
     </widget>
     <widget class="QListView" name="lsvSourceFiles">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>371</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeGroups">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>401</width>
        <height>501</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabIncludes">
     <attribute name="title">
      <string>Include headers</string>
     </attribute>
     <widget class="QListView" name="lsvIncludeHeaders">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>781</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-1</y>
        <width>781</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Includes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDefines">
     <attribute name="title">
      <string>Defines</string>
     </attribute>
     <widget class="QListView" name="lsvDefines">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>781</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>771</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Defines</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDependencies">
     <attribute name="title">
      <string>Dependencies</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Repo"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Repo">
   <property name="text">
    <string>Open Repo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen_Repo</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnOpenRepo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnOpenRepo_clicked()</slot>
 </slots>
</ui>
